google/cloud/spanner_v1/services/spanner/async_client.py

```diff
@@ -1045,8 +1045,8 @@ async def begin_transaction(
 
         if session is not None:
             request.session = session
-        if options is not None:
-            request.options = options
+        if options_ is not None:
```

Comment:
Please ignore this PR, tracking a revert in https://github.com/googleapis/gapic-generator-python/issues/835